src/patterns/atoms/links/text.hbs

```diff
@@ -14,6 +14,7 @@ tips: |
 nonos: |
   - Don’t use “click here” as link text.
   - Don‘t use text links (the `a` element) to trigger scripted actions unless the link also has a real destination in case JavaScript isn’t available. The `href` attribute should have a real URL, not `#` or `javascript:void(0)`. For script-only actions use a `button` element. Buttons do things; links go places.
+  - Dont't use rel="nofollow" for internal links
```

Comment:
Let's practice updating a pull request, by changing `Don't` to `Don’t` (with a curly apostrophe)

Reply:
Will give it a try later. Thanks!